### PR TITLE
Fix hanging bug

### PR DESCRIPTION
**Pull request by FaustXVI**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -207,25 +208,34 @@
...
                                     <value>COVEREDRATIO</value>
-                                    <minimum>1.00</minimum>
+                                    <!-- coverage is not at 100% because of jacoco try with resource problem -->
+                                    <minimum>0.9</minimum>
                                 </limit>
                                 <limit>
                                     <counter>COMPLEXITY</counter>
                                     <value>MISSEDCOUNT</value>
-                                    <maximum>0</maximum>
+                                    <!-- coverage is not at 100% because of jacoco try with resource problem -->
```
Comment (vdemeester) [inline pom.xml:237]: This comment feels weird 😅

File: src/main/java/com/github/junit5docker/DockerExtension.java [hunk 2]
```diff
@@ -74,7 +75,11 @@ private void waitForLogAccordingTo(WaitFor waitFor) {
     }
 
     private Supplier<Boolean> findFirstLogContaining(String logToFind) {
-        return () -> dockerClient.logs(containerId).anyMatch(log -> log.contains(logToFind));
+        return () -> {
+            try (Stream<String> logs = dockerClient.logs(containerId)) {
```
Comment (vdemeester) [inline src/main/java/com/github/junit5docker/DockerExtension.java:79]: Should we comment about that one (`AutoCloseable` feature right ?) ?

**Review by vdemeester — APPROVED** [review]
LGTM 🦁 
Just few questions 👼

File: pom.xml (hunk 1 above)
Comment (FaustXVI) [inline pom.xml:237]: Yes. I hope to be able to remove it later...

File: src/main/java/com/github/junit5docker/DockerExtension.java (hunk 2 above)
Comment (FaustXVI) [inline src/main/java/com/github/junit5docker/DockerExtension.java:79]: That's a language feature. I don't think we need a comment. The expected behaviour is covered by a test.

Comment (vdemeester) [reply src/main/java/com/github/junit5docker/DockerExtension.java:79]: @FaustXVI right, but I had to look it up 😛 
It's your call though 👼

**Pull request merged**